### ISSUE TITLE
Korean ancestry

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10927]
**Preferred term label:**
Korean ancestry

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK499913/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
ancestry, ethnicity, etc., is outside of the scope of HPO, because do model it well would require a separate framework.

**Comment by MickeySegal** [comment]
I've now added an "Outside of Scope" indicator for our findings so as to record which findings are out of scope for HPO and thus avoid flagging them in the future in a review of findings without HPO codes.
